website/content/docs/connect/config-entries/service-router.mdx

```diff
@@ -1,55 +1,877 @@
+| `Invert`  | Specifies that the logic for the HTTP header match should be inverted. Requests with matching criteria are not routed. | Boolean | `false` |
+
+### `Routes[].Match{}.HTTP{}.QueryParam`
+
+Specifies information to match to on HTTP query parameters. When more than one field is configured, all criteria must match for the service routing to apply.
+
+When using this field, do not configure `Present`, `Exact`, and `Regex` in a single map. You can use only one of these fields at a time when configuring match criteria for HTTP query parameters, as they are mutually exclusive with one another.
+
+#### Values
+
+- Default: None
+- Data type: List of maps. Each map can contain one or more of the following parameters:
+
+| Parameter | Description                                                                                                                            | Data type | Default |
+| :-------- | :------------------------------------------------------------------------------------------------------------------------------------- | --------- | ------- |
+| `Name`    | Specifies the name of the HTTP query parameter to match. This value is required when using [`Routes[].Match{}.HTTP{}.QueryParam`](#routes-match-http-queryparam). | String | None |
+| `Present` | Specifies that a request matches when the value in the `Name` field is present anywhere in the HTTP query parameter. When using this field, do not configure `Exact` or `Regex` in the same map. | Boolean | `false` |
+| `Exact`   | Specifies that a request matches when the query parameter with the given name is this exact value. When using this field, do not configure `Present` or `Regex` in the same map. | String | None |
+| `Regex`   | Specifies that a request matches when the query parameter with the given name matches this regular expression. When using this field, do not configure `Present` or `Exact` in the same map. The syntax for the regular expression field is proxy-specific. When [using Envoy](/consul/docs/connect/proxies/envoy), refer to [the documentation for Envoy v1.11.2 or newer](https://github.com/google/re2/wiki/Syntax) or [the documentation for Envoy v1.11.1 or older](https://en.cppreference.com/w/cpp/regex/ecmascript), depending on the version of Envoy you use. | String | None |
+
+### `Routes[].Destination`
+
+Specifies the target service to route matching requests to, as well as behavior for the request to follow when routed.
+
+#### Values
+
+- Default: None
+- Data type: Map containing one or more of the following parameters:
+
+| Parameter | Description                                                                                                                            | Data type | Default |
```

Comment:
```suggestion
| Parameter | Description                                                                                                                            | Data type | Default |
```
I think these params are better served as their own entries in the specification. Including the reason I mentioned in another comment about the control+f workflow, it also breaks the configuration model. We tell readers to click on a parameter to view it's details and the links to these parameters don't work. We could just link them all to #routes-destination, but I think it's a little counter to the spirit of the format we're trying to establish. For lower-nested configurations, it totally makes sense to place them into a table and link to the parent param, as in the  `Add`, `Set`, `Remove` maps in the `RequestHeaders` and `ResponseHeaders` fields. But I think this is too much of a deviation from the format.